components/odh-notebook-controller/controllers/notebook_controller.go

```diff
@@ -502,7 +504,7 @@ func (r *OpenshiftNotebookReconciler) SetupWithManager(mgr ctrl.Manager) error {
 
 				// If the ConfigMap is odh-trusted-ca-bundle or kube-root-ca.crt
```

Comment:
Optional: this comment is out of date now:
```suggestion
				// If the ConfigMap name matches on of our interested ConfigMaps
```

Reply:
Thanks for the comment, updated it.